### PR TITLE
Use reflection for PointerBufferPoolMXBean to avoid warnings on Android

### PR DESCRIPTION
**Pull request by tmm1**
cc https://github.com/bytedeco/javacpp/issues/446#issuecomment-752294063

### CODE REVIEW

File: src/main/java/org/bytedeco/javacpp/Pointer.java [hunk 1]
```diff
@@ -512,7 +511,13 @@ public static long parseBytes(String string, long relativeMultiple) throws Numbe
 
         String mx = System.getProperty("org.bytedeco.javacpp.mxbean", "false").toLowerCase();
         if (mx.equals("true") || mx.equals("t") || mx.equals("")) {
-            PointerBufferPoolMXBean.register();
+            try {
+                Class c = Class.forName("PointerBufferPoolMXBean");
```
Comment (tmm1) [inline src/main/java/org/bytedeco/javacpp/Pointer.java:515]: Does this need to be fully qualified?

```suggestion
                Class c = Class.forName("org.bytedeco.javacpp.tools.PointerBufferPoolMXBean");
```

Comment (saudet) [reply src/main/java/org/bytedeco/javacpp/Pointer.java:515]: Yes, it won't work otherwise.

File: src/main/java/org/bytedeco/javacpp/Pointer.java [hunk 2]
```diff
@@ -512,7 +511,13 @@ public static long parseBytes(String string, long relativeMultiple) throws Numbe
...
+                Method method = c.getDeclaredMethod("register");
+                method.invoke(null);
+            } catch (Throwable t) {
```
Comment (saudet) [inline src/main/java/org/bytedeco/javacpp/Pointer.java:518]: ReflectiveOperationException would probably suffice here.

Comment (tmm1) [reply src/main/java/org/bytedeco/javacpp/Pointer.java:518]:
```suggestion
            } catch (ReflectiveOperationException t) {
```

**Review by matteodg — APPROVED** [review]

**Pull request merged**